### PR TITLE
Adds silver, bronze, and pewter trophies

### PR DESCRIPTION
**Pull request by Incoming5643**
so you can properly quantify a player's level of success

glode silvnize bronize pewnize

Removes the subtyped trophy that has no reason to exist in a post 2011 world.

### CODE REVIEW

File: code/modules/food&drinks/drinks/drinks.dm [hunk 1]
```diff
@@ -95,9 +95,9 @@
 /// Drinks. END
 ////////////////////////////////////////////////////////////////////////////////
 
-/obj/item/weapon/reagent_containers/food/drinks/golden_cup
-	desc = "A golden cup"
-	name = "golden cup"
+/obj/item/weapon/reagent_containers/food/drinks/gold_cup
+	name = "gold cup"
```
Comment (Cheridan) [inline code/modules/food&drinks/drinks/drinks.dm]: :+1:

File: code/modules/food&drinks/drinks/drinks.dm [hunk 2]
```diff
@@ -95,9 +95,9 @@
...
+	desc = "You're winner!"
```
Comment (Cheridan) [inline code/modules/food&drinks/drinks/drinks.dm]: :+1:

File: code/modules/food&drinks/drinks/drinks.dm [hunk 3]
```diff
@@ -109,10 +109,49 @@
 	flags = CONDUCT | OPENCONTAINER
 	spillable = 1
 
-/obj/item/weapon/reagent_containers/food/drinks/golden_cup/tournament_26_06_2011
-	desc = "A golden cup. It will be presented to a winner of tournament 26 june and name of the winner will be graved on it."
```
Comment (PKPenguin321) [inline code/modules/food&drinks/drinks/drinks.dm]: ;-;7

Comment (Iamgoofball) [inline code/modules/food&drinks/drinks/drinks.dm]: ;_;7

File: code/modules/food&drinks/drinks/drinks.dm [hunk 4]
```diff
@@ -109,10 +109,49 @@
...
+	icon_state = "pewter_cup"
+	w_class = 1
+	force = 1
+	throwforce = 1
+	amount_per_transfer_from_this = 5
+	materials = list(MAT_METAL=100)
+	possible_transfer_amounts = list()
+	volume = 5
+	flags = CONDUCT | OPENCONTAINER
+	spillable = 1
```
Comment (KorPhaeron) [inline code/modules/food&drinks/drinks/drinks.dm]: Shouldn't these all be child objects of the gold cup if they share the spillable/flags/possible transfer amounts?

I know it's incredibly nitpicky and will literally never matter but I don't want someone else to get grumpy at me if I merge it

**Comment by Incoming5643** [comment]
made the pewter cup the default /trophy/ and the other better cups subtypes of said cup

**Comment by KorPhaeron** [comment]
Thanks

**Pull request merged**